span-timeseries-transformer/src/main/resources/config/base.conf

```diff
@@ -24,10 +24,8 @@ kafka {
 
 haystack.graphite.host = "monitoring-influxdb-graphite.kube-system.svc"
 
-enable.metricpoint.period.replacement=true
+metricpoint.encoder.type = "periodreplacement"
```

Comment:
can we document the other encoder type available in the configuration ?

Reply:
👍 pushed